apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx

```diff
@@ -28,11 +28,10 @@ import NavItem from '../nav-item';
 import { Post } from '../../types';
 import './style.scss';
 
-const Button = ( {
-	children,
-	...rest
-}: OriginalButton.Props & { icon?: any; iconSize?: number } ) => (
-	<OriginalButton { ...rest }>{ children }</OriginalButton>
+const Button = forwardRef(
+	( { children, ...rest }: OriginalButton.Props & { icon?: any; iconSize?: number } ) => (
```

Comment:
```suggestion
	( { children, ...rest }: OriginalButton.Props & { icon?: any; iconSize?: number }, ref ) => (
```

Adding `ref` here. Maybe I can do it even better ...